glogger/sender.py

```diff
@@ -38,6 +38,9 @@ def __init__(
         server_address: str,
         *,
         scheme: str = "https",
+        x_auth_type: str = "",
```

Comment:
remove the commas else it will change the type into a tupple

Reply:
You must be confused.